maint_tools/check_gha_workflow.py

```diff
@@ -0,0 +1,263 @@
+"""Collects and plot run time of jobs in a GHA workflow.
+
+Requires:
+- requests
+- rich
+- pandas
+- plotly
+- kaleido
```

Comment:
Where is kaleido used in the script ?

Reply:
We need it to save the plotly images to disk.

Reply:
Got it, I see now that newer plotly versions (4.9+) will automatically use it when installed, but could we make that explicit ? So, set `engine="kaleido"`. I can add a suggestion below.